vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
NOTE: changed file name...lost most of Nathan's comments but applied all of them here.